src/services/resubmitCourtCase/index.ts

```diff
@@ -1 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
I thought I set this to warning too 🤔

Reply:
Not sure tbh. My concern with warnings is that it is easy to miss them. We might end up in an inconsistent situation where some modules are exported as default and some are not.